bci_main.py

```diff
@@ -4,14 +4,14 @@
 from bcipy.helpers.task import print_message
 from bcipy.helpers.system_utils import get_system_info, configure_logger
 from bcipy.helpers.language_model import init_language_model
-from bcipy.helpers.load import load_signal_model
+from bcipy.helpers.load import load_json_parameters, load_signal_model
 from bcipy.helpers.parameters import DEFAULT_PARAMETERS_PATH
 from bcipy.helpers.save import init_save_data_structure, DEFAULT_EXPERIMENT_ID
 from bcipy.tasks.start_task import start_task
 from bcipy.tasks.task_registry import ExperimentType
 
 
-def bci_main(parameter_location: str, user: str, exp_type: int, mode: str, experiment: str = DEFAULT_EXPERIMENT_ID) -> bool:
+def bci_main(parameter_location: str, user: str, task: int, experiment: str = DEFAULT_EXPERIMENT_ID) -> bool:
```

Comment:
Looks like the type annotation for the `task` parameter is inconsistent with the documentation.